packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1411,7 +1417,7 @@ class _TabBarViewState extends State<TabBarView> {
     super.didChangeDependencies();
     _updateTabController();
     _currentIndex = _controller!.index;
-    _pageController = PageController(initialPage: _currentIndex!);
+    _pageController = PageController(initialPage: _currentIndex!, viewportFraction: widget.viewportFraction);
```

Comment:
Style guide nit: more than one parameter should be split out over multiple lines
```suggestion
    _pageController = PageController(
      initialPage: _currentIndex!,
      viewportFraction: widget.viewportFraction,
    );
```